db/migrate/20150407143903_create_urls.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUrls < ActiveRecord::Migration
+  def change
+    create_table :urls do |t|
+      t.string :short, null: false
```

Comment:
As a general rule, you should have indexes on any column which you look up objects by, so I would put indices on `short` and `long`.